server/graphql/v2/mutation/PaymentMethodMutations.js

```diff
@@ -0,0 +1,59 @@
+import { GraphQLEnumType, GraphQLInt, GraphQLNonNull, GraphQLString } from 'graphql';
+
+import models from '../../../models';
+import { setupCreditCard } from '../../../paymentProviders/stripe/creditcard';
+import { NotFound, Unauthorized } from '../../errors';
+import { PaymentMethodCreateInput } from '../input/PaymentMethodCreateInput';
+import { PaymentMethod } from '../object/PaymentMethod';
+
+const paymentMethodMutations = {
+  addStripeCreditCard: {
+    type: PaymentMethod,
+    description: 'Add a new payment method to be used with an Order',
+    args: {
+      newPaymentMethod: {
+        type: PaymentMethodCreateInput,
+        description: 'Reference to a Payment Method to add to an Account',
+      },
```

Comment:
We're still missing an account (`AccountReference`) to know to which account we should add the Credit Card.